zhaquirks/mli/tintE14rgbcct.py

```diff
@@ -0,0 +1,97 @@
+"""Tint E14 RGB CCT."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class TintManufacturerSpecificCluster(CustomCluster):
+    """Tint manufacturer specific cluster."""
+
+    cluster_id = 0x100F  # not inside the manufacturer specific cluster range
+
+
+class TintRGBCCTColorCluster(CustomCluster, Color):
+    """Tint RGB+CCT Lighting custom cluster."""
+
+    # Set correct capabilities to ct, xy, hs
+    # Tint bulbs do not correctly report this attribute
+    _CONSTANT_ATTRIBUTES = {0x400A: 0b11111}
```

Comment:
One more thing: did you check if the bulb actually supports Hue/Saturation, enhanced Hue, color loop, (XY, and color temp)?

You can test this by turning **off** the "Always prefer XY color mode" option in the ZHA config panel:
<img src="https://user-images.githubusercontent.com/6409465/212362445-5263e1d0-eb54-4163-a8d5-9eb3d41ca5ff.png" width="400">
This will make ZHA use "enhanced Hue" commands if supported, otherwise it'll use normal HS commands, or fallback to XY.
Because of this of quirk, ZHA should use "enhanced Hue" commands. If that doesn't work, change this to ``0b11110``. If that also doesn't work, change it to ``0b11100``.
Please also test if the "color loop" effect works with this light.


Judging [from this](https://github.com/zigpy/zigpy/blob/297a2b0f364fe7a31de3b4966d70d1a67d432358/zigpy/zcl/clusters/lighting.py#L25-L30), it should be:
``0b11000`` if only XY and color temp are supported
``0b11100`` if only XY, color temp, and color loop are supported
``0b11110`` if only XY, color temp, color loop, and HS are supported (but not "enhanced Hue")
... and so on

Other than that, the PR looks ready to me!

Reply:
I'll check all these options and update the capabilities

Reply:
After turning off "Always prefer XY color mode":
with `0b11111` the light would turn red when I tried to set any color
with `0b11110` it worked properly

Also I tested color loop and it works. 

So I decided to change it to `0b11110`